plugins/stackstorm/src/components/ActionsList/ActionsList.tsx

```diff
@@ -0,0 +1,163 @@
+/*
+ * Copyright 2023 The Backstage Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+import React, { useState } from 'react';
+import useAsync from 'react-use/lib/useAsync';
+import { Progress } from '@backstage/core-components';
+import { useApi, configApiRef } from '@backstage/core-plugin-api';
+import {
+  List,
+  ListItemText,
+  Collapse,
+  ListItem,
+  ListItemSecondaryAction,
+  ListItemIcon,
+} from '@material-ui/core';
+import { makeStyles } from '@material-ui/core/styles';
+import ExpandMore from '@material-ui/icons/ExpandMore';
+import ExpandLess from '@material-ui/icons/ExpandLess';
+import { Alert } from '@material-ui/lab';
+import { Action, Pack, stackStormApiRef } from '../../api';
+
+const useStyles = makeStyles(theme => ({
+  root: {
+    width: '100%',
+    backgroundColor: theme.palette.background.paper,
+  },
+  actions: {
+    borderBottom: `2px solid ${theme.palette.divider}`,
+  },
+  nested: {
+    paddingLeft: theme.spacing(8),
+    paddingRight: theme.spacing(4),
+  },
+  icon: {
+    minWidth: '34px',
+  },
+}));
+
+type ActionItemsProps = {
+  pack: Pack;
+};
+
+export const ActionItems = ({ pack }: ActionItemsProps) => {
+  const classes = useStyles();
+  const st2 = useApi(stackStormApiRef);
+  const config = useApi(configApiRef);
+
+  const { value, loading, error } = useAsync(async (): Promise<Action[]> => {
+    const data = await st2.getActions(pack.ref);
+    return data;
+  }, []);
+
+  if (loading) {
+    return <Progress />;
+  } else if (error) {
+    return <Alert severity="error">{error.message}</Alert>;
+  }
+
+  return (
+    <List component="div" disablePadding className={classes.actions}>
+      {(value || []).map(a => {
+        return (
+          <ListItem
+            key={a.ref}
+            button
+            className={classes.nested}
+            onClick={() =>
+              window.open(
```

Comment:
Would really prefer to not have window.open. Could make this into a proper Link behavior instead, using Link from `@backstage/core-components`:

```tsx
<ListItem
  component={Link}
  to={`${config.getString('stackstorm.webUrl')}/?#/actions/${a.ref}`}
  ...other props
```

Note that Link handles the _blank etc for you, along with security features such as noopener/noreferrer as needed, and in the future metrics tracking, etc